MapboxVision/Services/MotionManager.swift

```diff
@@ -2,30 +2,28 @@ import Foundation
 import CoreMotion
 import CoreLocation
 
+private let referenceFrame: CMAttitudeReferenceFrame = .xArbitraryZVertical
```

Comment:
Could you please extract const into a static let within case-less enum?

Reply:
done